From 1507709e3c7d71a23972acc84f496e9dc5a81caf Mon Sep 17 00:00:00 2001
From: Josh Wilsdon <josh@wilsdon.ca>
Date: Fri, 18 Jan 2019 14:45:34 -0800
Subject: [PATCH] TRITON-1082 hermes-proxy should not run itself out of files
 and memory Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 CHANGES.md   | 4 ++++
 deps/hermes  | 2 +-
 package.json | 2 +-
 3 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index 7411c35..16d80e7 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -1,5 +1,9 @@
 # sdc (SDC ops core zone) Changelog
 
+## 1.6.1
+
+- TRITON-1082 hermes-proxy should not run itself out of files and memory
+
 ## 1.6.0
 
 - AGENT-997 update hermes to sdcnode (v6.15.1)
diff --git a/deps/hermes b/deps/hermes
index 6744fa4..d3c3ea8 160000
--- a/deps/hermes
+++ b/deps/hermes
@@ -1 +1 @@
-Subproject commit 6744fa48829270d50ee8331a58f372b9aae5cc12
+Subproject commit d3c3ea81fffb0df23d22ede731d2e6bdd7384073
diff --git a/package.json b/package.json
index 84d5d12..ac9ac74 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "sdc",
     "description": "SmartDataCenter 'sdc' zone to hold ops/admin tools",
-    "version": "1.6.0",
+    "version": "1.6.1",
     "author": "Joyent (joyent.com)",
     "private": true,
     "dependencies": {
-- 
2.21.0

